test/e2e/config/azure-dev.yaml

```diff
@@ -92,19 +92,15 @@ providers:
       - sourcePath: "../data/infrastructure-azure/v1beta1/cluster-template.yaml"
         targetName: "cluster-template-management.yaml"
       - sourcePath: "../data/infrastructure-azure/v1beta1/cluster-template-kcp-adoption.yaml"
-        targetName: "cluster-template-kcp-adoption.yaml"
-      - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-ipv6.yaml"
-        targetName: "cluster-template-ipv6.yaml"
       - sourcePath: "../data/infrastructure-azure/v1beta1/cluster-template-md-remediation.yaml"
-        targetName: "cluster-template-md-remediation.yaml"
       - sourcePath: "../data/infrastructure-azure/v1beta1/cluster-template-kcp-remediation.yaml"
-        targetName: "cluster-template-kcp-remediation.yaml"
       - sourcePath: "../data/infrastructure-azure/v1beta1/cluster-template-kcp-scale-in.yaml"
-        targetName: "cluster-template-kcp-scale-in.yaml"
       - sourcePath: "../data/infrastructure-azure/v1beta1/cluster-template-node-drain.yaml"
-        targetName: "cluster-template-node-drain.yaml"
+      - sourcePath: "../data/infrastructure-azure/v1beta1/cluster-template-upgrades.yaml"
       - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-machine-pool.yaml"
         targetName: "cluster-template-machine-pool.yaml"
+      - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-ipv6.yaml"
+        targetName: "cluster-template-ipv6.yaml"
```

Comment:
...but `targetName` is needed here because...`sourcePath` isn't relative? Or something? I'm mildly confused by this cleanup.

Reply:
yeah basically `targetName` is only needed if we want to rename the file, otherwise it just uses the filename. In this case we want to remove the "prow" part from the file name so it can be used as the "ipv6" flavor.